src/CoreGraphics/CGColorSpace.cs

```diff
@@ -378,6 +379,50 @@ public NSData GetICCProfile ()
 			IntPtr ptr = CGColorSpaceCopyICCProfile (handle);
 			return (ptr == IntPtr.Zero) ? null : new NSData (ptr, true);
 		}
+
+		[iOS (10,0)][Mac (10,12)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern /* CFDataRef* */ IntPtr CGColorSpaceCopyICCData (/* CGColorSpaceRef */ IntPtr space);
+
+		[iOS (10,0)][Mac (10,12)]
+		public NSData GetIccData ()
+		{
+			IntPtr ptr = CGColorSpaceCopyICCData (handle);
+			return (ptr == IntPtr.Zero) ? null : new NSData (ptr, true);
+		}
+
+		[iOS (10,0)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern unsafe /* CFStringRef* */ IntPtr CGColorSpaceCopyName (/* CGColorSpaceRef */ IntPtr space);
+
+		[iOS (10,0)]
+		public string Name {
+			get {
+				return CFString.FetchString (CGColorSpaceCopyName (handle));
```

Comment:
What happens if `handle == IntPtr.Zero`? Should we throw an `ObjectDisposedException` here?